supertokens_python/recipe/thirdpartypasswordless/interfaces.py

```diff
@@ -8,80 +8,33 @@
     SignInUpPostOkResponse, SignInUpPostNoEmailGivenByProviderResponse,
     SignInUpPostFieldErrorResponse)
 from supertokens_python.recipe.thirdparty.provider import Provider
-from typing_extensions import Literal
 
 from ..passwordless import interfaces as PlessInterfaces
-from ..passwordless.interfaces import (CreateCodePostResponse,
-                                       CreateCodeResult,
-                                       CreateNewCodeForDeviceResult,
```

Comment:
You need to duplicate all the types that have the `User` type in them cause if you don't, then the user type being returned for different function calls is different.

(Same thing is applied to thirdpartyemailpassword recipe as well)